opus/import/main_opus_import.py

```diff
@@ -298,6 +298,11 @@
     help='Omit tracebacks from exception reports'
 )
 
+# Argument about whether to check index row files or not
+parser.add_argument(
+    '--import-use-row-files', action='store_true', default=False,
+    help='Import with checking row files under shelves/index'
```

Comment:
Let's make the help a little more user-friendly. Something like "Use metadata row files to determine whether index and summary files should be included in the files table".